src/compiler/checker.ts

```diff
@@ -113,6 +116,8 @@ namespace ts {
         let globalRegExpType: ObjectType;
         let globalTemplateStringsArrayType: ObjectType;
         let globalESSymbolType: ObjectType;
+        let jsxElementType: ObjectType;
+        let jsxIntrinsicElementsType: ObjectType;

```

Comment:
Are any of the other ones lazily-loaded? Can you put a comment above like

```TypeScript
/**
 * DO NOT USE DIRECTLY.
 *
 * @see getJsxIntrinsicElementsType.
 */
```

or something like that, depending on if `@see` is a valid tag name.